### ISSUE TITLE
Mobile (?) Changer l'ordonnancement des propositions n'est pas possible (?) 

### ISSUE THREAD

**Issue by TheoSabattie** [MieuxVoter/majority-judgment-web-app#96]
### Issue and Steps to Reproduce

Je suis parvenu à drag'n'drop les mentions pour changer l'ordre mais cel ne m'a pas semblé possible avec les propositions. M'y suis-je mal pris ? Ou est-ce une feature non developpée ?

**Comment by TheoSabattie** [comment]
![Screenshot_20221206_072735_com android chrome](https://user-images.githubusercontent.com/2721436/205837796-d14eae15-a42b-4d36-8359-873cf28bcf5c.jpg)

**Comment by TheoSabattie** [comment]
![Screenshot_20221206_072935_com android chrome](https://user-images.githubusercontent.com/2721436/205837965-03df37e4-4b72-4eaf-a6fd-966735831826.jpg)

À noter: devrait être reordonable aux deux endroits spécifiés en screenshots

**Comment by guhur** [comment]
Right, I will add this feature. Since ordering is not necessarily random anymore, it is an important feature. Thanks!

**Comment by guhur** [comment]
3cd3757 adds this feature!
Thanks a lot for your suggestion :hugs:
